src/lib/Input.svelte

```diff
@@ -69,7 +76,7 @@
     <button class="send" disabled={!canSend} on:click={send}>
         <span class="material-symbols-outlined">send</span>
     </button>
-    <button class="clear">
+    <button class="clear" on:click={resetStore}>
```

Comment:
Nice, thanks for adding this! 

Not for this PR, but something will need to discuss with both the Chat Tutor folks and the researchers is how old messages should be included when prompting the `/ask` endpoint. Currently, all messages are sent along with the request, and it's effectively the client that keeps track of message history (this also relates to the local storage ticket). Something kind of nice about this is that a user can control the 'context' the tutor is sent by clearing there messages with this button. However, I'm not sure if that will work for the researchers

Reply:
Sure.